source/_integrations/panel_iframe.markdown

```diff
@@ -1,8 +1,8 @@
 ---
-title: iframe Panel
+title: iframe panel
```

Comment:
```suggestion
title: iFrame panel
```
MS uses `iframe`, `iFrame` and `IFRAME` for this, not sure which is the correct one.
but `iFrame` matches the description.